dev/js/wow_wizard.js

```diff
@@ -378,6 +378,30 @@
 			}
 		}
 
+		// If the step has steps depending on it, shows alert and clear the future steps.
+		function _clearFutureSteps($step) {
+			var depending_steps = [];
+			
+			for(var i = currentStep+1; i <= passedStepTracker; i++) {
+				var step_filled = wizard.settings.steps[i];
+				if(step_filled.isDependent) {
+					var found = false;
+					for(var j = 0; j < step_filled.steps.length && !found; j++) {
+						if(step_filled.steps[j].triggerStep == currentStep) {
+							depending_steps.push(i);
+							found = true;
+						}
+					}
+				}
+			}
+			if(depending_steps.length > 0) {
+				var message = (depending_steps.length == 1)?'Step ':'Steps ';
+				message += depending_steps + ' will change.'; 
+				alert(message);
+				passedStepTracker = depending_steps[0];
```

Comment:
Yep, you are right, that makes sense. Looks better now, thanks! After testing on my local, I will merge this. 
Also one more note: we should probably let developer who uses this plugin activate and deactivate these alert messages, some may find these annoying; but that will be another issue itself.